packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.tsx

```diff
@@ -110,14 +110,17 @@ export class CommandBar extends BaseComponent<ICommandBarProps, ICommandBarState
         </div>
       );
     }
+    // Total # of menu items is regular items + far items + 1 for the ellipsis, if necessary
+    const setSize = renderedItems!.length + renderedFarItems!.length + (renderedOverflowItems && renderedOverflowItems.length > 0 ? 1 : 0);
+    let posInSet = 1;
 
     return (
       <div className={ css('ms-CommandBar', styles.root, className) } ref='commandBarRegion'>
         { searchBox }
         <FocusZone ref='focusZone' className={ styles.container } direction={ FocusZoneDirection.horizontal } role='menubar' >
           <div className={ css('ms-CommandBar-primaryCommands', styles.primaryCommands) } ref='commandSurface'>
-            { renderedItems!.map((item, index) => (
-              this._renderItemInCommandBar(item, index, expandedMenuItemKey!)
+            { renderedItems!.map(item => (
+              this._renderItemInCommandBar(item, posInSet++, setSize, expandedMenuItemKey!)
```

Comment:
>posInSet [](start = 49, length = 8)

Can you please add another value to the snapshot tests such that the count increment is working correctly. Or a unit test would be equally good.

Reply:
Not sure what you meant by "add another value to the snapshot tests such that the count increment is working correctly" so I added a unit test.

Reply:
That works! But he just meant add another item in the `items` of the command bar in the snapshot unit test which is easier to at least validate in the snapshot that the setSize works. :)

Reply:
Ok, thanks David!